analytics/lib/counts.py

```diff
@@ -177,7 +177,9 @@ def do_aggregate_to_summary_table(stat, end_time):
         start = time.time()
         cursor.execute(realmcount_query, {'end_time': end_time})
         end = time.time()
-        logger.info("%s RealmCount aggregation (%dms/%sr)" % (stat.property, (end-start)*1000, cursor.rowcount))
+        logger.info("%s RealmCount aggregation (%dms/%sr)" % (stat.property,
+                                                              (end-start)*1000,
+                                                              cursor.rowcount))
```

Comment:
These would be better wrapped with the newline after the `(`.

Reply:
Fixed.